examples/app_commands/slash_options.py

```diff
@@ -29,5 +31,18 @@ async def channel(
     await ctx.respond(f"Hi! You selected {channel.mention} channel.")
 
 
+@bot.slash_command(name="attach_file")
+async def say(
+    ctx: discord.ApplicationContext,
+    attachment: Option(
+        SlashCommandOptionType.attachment,

```

Comment:
Using `discord.Attachment` here would be more consistent with the rest of the example

Reply:
i agree